include/volume/volume_cooling_gaussians.hpp

```diff
@@ -457,7 +455,7 @@ double volume_cooling_gaussians(Polytope& Pin,
 
 template
 <
-    typename WalkTypePolicy = GaussianCDHRWalk,
```

Comment:
Please don't change the default random walk in `volume_cooling_gaussians<>`.
You can create a new example in folder `examples` that it will call this function templated with `GaussianHamiltonianMonteCarloExactWalk`. You can write 3-4 examples that compute volumes of polytopes from 10 to 50 dimensions. You can use the volesti polytope generators.